docker-compose.yml

```diff
@@ -42,15 +43,16 @@ services:
     ports:
       - "3306:3306"
 
-  nodejs:
+  node:
     build:
       context: .
-      target: sylius_nodejs
-      cache_from:
-        - quay.io/sylius/php:latest
-        - quay.io/sylius/nodejs:latest
-        - quay.io/sylius/nginx:latest
-    image: quay.io/sylius/nodejs:latest
+      target: sylius_node
+      # Quay does not work, should be replaced in future with f.e. ghcr.io
+      # cache_from:
+      #   - quay.io/sylius/php:latest
+      #   - quay.io/sylius/nodejs:latest
+      #   - quay.io/sylius/nginx:latest
```

Comment:
Commented config (unless left for reference for users) should be removed - as it is just a dead "code"